tests/pgkit_test.go

```diff
@@ -349,6 +351,35 @@ func TestRowsWithBigInt(t *testing.T) {
 		assert.NoError(t, err)
 		assert.Equal(t, "count2", sout.Key)
 		assert.True(t, sout.Num.String() == "12323942398472837489234")
+
+		pretty.Println(sout.Rating)
+
+		assert.Nil(t, sout.Rating)
+	}
+
+	// last, with opt rating
+	{
+		bv := dbtype.NewBigInt(5)
+
+		stat := &Stat{
+			Key:    "count3",
+			Num:    dbtype.NewBigIntFromString("44", 0),
+			Rating: &bv,
```

Comment:
passing *dbtype.BigInt here is causing an assignment issue to a nullable NUMERIC(78,0) type